SPECS/cloud-init-vmware-guestinfo/cloud-init-vmware-guestinfo.spec

```diff
@@ -0,0 +1,41 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+Name:           cloud-init-vmware-guestinfo
+Version:        1.3.1
+Release:        1
+Summary:        A cloud-init datasource for VMware
+Group:          System/Management
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/vmware/cloud-init-vmware-guestinfo
+BuildArch:      noarch
+
+#Source0:  https://github.com/vmware/%{name}/archive/v%{version}.tar.gz
```

Comment:
>   [](start = 9, length = 2)

Nit: could you also align this comment's spacing to other tags and put `BuildArch` behind the sources to align better with other specs?